src/features/CourtCaseList/CourtCaseListEntry.tsx

```diff
@@ -0,0 +1,184 @@
+import ConditionalRender from "components/ConditionalRender"
+import DateTime from "components/DateTime"
+import { GridRow, Link, Table } from "govuk-react"
+import Image from "next/image"
+import { useRouter } from "next/router"
+import { encode } from "querystring"
+import { useState } from "react"
+import { createUseStyles } from "react-jss"
+import CourtCase from "services/entities/CourtCase"
+import User from "services/entities/User"
+import { deleteQueryParamsByName } from "utils/deleteQueryParam"
+import getTriggerWithDescription from "utils/formatReasons/getTriggerWithDescription"
+import groupErrorsFromReport from "utils/formatReasons/groupErrorsFromReport"
+import { displayedDateFormat } from "utils/formattedDate"
+import {
+  filterUserNotes,
+  first100CharsOfMostRecentNote,
+  getMostRecentNote,
+  validatedMostRecentNoteDate
+} from "./CourtCaseListEntryHelperFunction"
+import { NotePreview, NotePreviewButton } from "./NotePreviewButton"
+import LockedByTag from "./tags/LockedByTag"
+import ResolvedTag from "./tags/ResolvedTag"
+import UrgentTag from "./tags/UrgentTag"
+
+const useStyles = createUseStyles({
+  caseDetailsRow: {
+    verticalAlign: "top",
+    borderColor: "unset"
+  },
+  triggersRow: {
+    verticalAlign: "top",
+    backgroundColor: "#f3f2f1" // GDS light-grey color
+  },
+  flexBox: {
```

Comment:
`flexBox` is quite generic & could be used elsewhere, so we could move it to `styles/customStyles.ts`, so that it can be reused. Personally, I'm not fussed about it. It's just a suggestion.

Reply:
Interested to hear @C-gyorfi  @monotypical , these are custom JSS to a component in this file and it is useful when developing to see the JSS styles within the same file. 

Is there much value abstracting this out into a separate file?

Reply:
I'd say its valiable if we use it elsewhere